src/App.jsx

```
@@ -1,58 +1,62 @@
 import { Route, Routes, Navigate } from "react-router-dom";
 import { AuthProvider } from "./context/AuthContext";
+import { Helmet } from "react-helmet-async";
+
+import Home from "./User/pages/Home";
+import UserAuth from "./User/pages/UserAuth";
+import Privacy from "./User/pages/Privacy-Policy";
+import TermsAndConditions from "./User/pages/TermsAndCondition";
+
+
+// User pages (FIXED IMPORTS)
+import UserAuth from "./User/pages/UserAuth";
```

Biome (2.3.13)

[error] Shouldn't redeclare 'UserAuth'. Consider to delete it or rename it.

'UserAuth' is defined here:


(lint/suspicious/noRedeclare)